qiskit/assembler/disassemble.py

```diff
@@ -152,6 +178,18 @@ def _experiments_to_circuits(qobj):
             if conditional and name != "bfunc":
                 _inst.c_if(conditional["register"], conditional["value"])
                 conditional = {}
+        pulse_lib = qobj.config.pulse_library if hasattr(qobj.config, "pulse_library") else []
+        parametric_pulses = (
+            qobj.config.parametric_pulses if hasattr(qobj.config, "parametric_pulses") else []
+        )
+        if hasattr(qobj.config, "calibrations"):
+            circuit.calibrations = dict(
+                **circuit.calibrations, **_qobj_to_circuit_cals(qobj, pulse_lib, parametric_pulses)
```

Comment:
Is it possible for `circuit.calibrations` to have a value already?

Reply:
`circuit.calibrations={}` before L186 of the first iteration of the for-loop, and can be valued after L186 and at every subsequent iteration (correct me if im wrong)